extension/js/common/composer.ts

```diff
@@ -1543,7 +1544,7 @@ export class Composer {
       }
     }, this.getErrHandlers(`focus on recipient field`))).children().click(() => false);
     this.resizeInputTo();
-    this.attach.initAttDialog('fineuploader', 'fineuploader_button');
+    this.attach.initAttDialog('fineuploader', 'fineuploader_button', async () => this.setInputTextHeightManuallyIfNeeded());
```

Comment:
Would it be possible to re-use `this.attach.setAttAddedCb(...)` ? That way we would not need to add a new `onSuccess` callback above.

Reply:
oh, yes, I didn't see that method.